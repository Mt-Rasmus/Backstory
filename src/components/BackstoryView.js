
import React from 'react';

const BackstoryView = (props) => {
   return (
      <div>
         {props.backstory}
         <button 
            className="button button--standard button-modal" 
            onClick={props.toggleShowBackstory}>
            Premise
         </button>
         <input type="checkbox" id="vehicle1" onClick={props.toggleSolved}></input>
         <label htmlFor="vehicle1">solved</label><br></br>            
      </div>
   )
}

export default BackstoryView