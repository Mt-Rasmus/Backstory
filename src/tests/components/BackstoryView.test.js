
import React from 'react';
import BackstoryView from '../../components/BackstoryView';
import { shallow } from 'enzyme';

test('Should render BackstoryView correctly', () => {
   const wrapper = shallow(<BackstoryView />);
   expect(wrapper).toMatchSnapshot();
});
