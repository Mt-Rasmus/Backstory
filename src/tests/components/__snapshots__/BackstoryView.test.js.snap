// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render BackstoryView correctly 1`] = `
<div>
  <button
    className="button button--standard button-modal"
  >
    Premise
  </button>
  <input
    id="vehicle1"
    type="checkbox"
  />
  <label
    htmlFor="vehicle1"
  >
    solved
  </label>
  <br />
</div>
`;
